packages/dashboard/src/pages/routes.tsx

```diff
@@ -39,11 +40,17 @@ export function Pages() {
     useEffect(() => {
         return Messages.events.restoreSuccess.on(restoreCallback)
     }, [restoreCallback])
-
     return (
         <Suspense fallback={null}>
             <Routes>
-                <Route path={DashboardRoutes.Welcome} element={<Welcome />} />
+                <Route
+                    path={DashboardRoutes.Welcome}
+                    element={
+                        <NoPersonaGuardRoute redirectTo={DashboardRoutes.Personas}>
```

Comment:
is this change breaks https://github.com/DimensionDev/Maskbook/pull/8328?

Reply:
No, it shouldn't be removed in #8328. Removing it is for bypassing.